src/components/Card.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+
+type CardProps = {
+  cardData: PokemonCardObject
+  isCollected: boolean
```

Comment:
you already have the cardData and collection, I'd just add that logic down here instead of calculating it above and prop-drilling.